### ISSUE TITLE
Increase loss query in hot flip attack

### ISSUE THREAD

**Issue by anubhakabra** [Eric-Wallace/universal-triggers#11]
In the **attacks.py** file, line number 30 (hot flip attack), according to the commented part of the code (line 19), shouldn't it be multiplying the gradient dot embedding matrix with -1 when increase loss is True?
hence line 30 should be [ `**if increase_loss :**` ] ?

**Comment by Eric-Wallace** [comment]
The comment might be a little misleading. Then main missing point is that there is also a negative sign coming from the loss function (cross entropy, so negative log likelihood).

If not ‘increase loss’, we multiply the grad by negative 1. This causes the gradient to tell us how to increase the log likelihood, which will decrease the original loss. Hope this helps!
